src/Hosting/Certificates/InitializationCertificateReader.cs

```diff
@@ -11,6 +12,7 @@ namespace Microsoft.Omex.Extensions.Hosting.Certificates
 	/// <remarks>
 	/// Please prefer using instance resolved from DI container.
 	/// </remarks>
+	[Obsolete("InitializationLogger using OmexLogger is obsolete and is pending for removal by 1 July 2024.", DiagnosticId = "OMEX188")]
```

Comment:
What is going to replace this class?

Reply:
Nothing for now. We can get this supported by OmexOpenTelemetry in the future. But for now it is obsolete. 
A workaround / renovation can come in another PR

Reply:
OTel does not provide a certificate reader class AFAIK.

Reply:
Yes, but we can have OpenTelemetryInitializationLogger here to build this InitializationCertificateReader

Reply:
If you are not going to delete this class, don't mark it obsolete.

Reply:
Yes, obsolete removed. Used Pragma instead